App/SB/views/FeedList/index.js

```diff
@@ -110,6 +111,23 @@ class Notifications extends React.PureComponent {
     )
   }
 
+  _renderPlaceholder() {
+    if (this.props.notifications.length === 0 && this.props.showTourScreen !== true) {
```

Comment:
Since loading from fresh shouldn't ever start with this screen (I think), no need to check for empty object like in thread view? Am I right in assuming this?